app/client/components/payment/update/card/stripeCardInputForm.tsx

```diff
@@ -34,18 +38,23 @@ interface StripeInputFormError {
 
 export const StripeCardInputForm = (props: StripeCardInputFormProps) => {
   const [isValidating, setIsValidating] = useState<boolean>(false);
-  const [readyElements, setReadyElements] = useState<string[]>([]);
+  const [cardNumberReady, markCardNumberReady] = useState<boolean>(false);
+  const [cardExpiryReady, markCardExpiryReady] = useState<boolean>(false);
+  const [cardCVCReady, markCardCVCReady] = useState<boolean>(false);
```

Comment:
Minor but implies boolean a bit better
```suggestion
  const [isCardNumberReady, markCardNumberReady] = useState<boolean>(false);
  const [isCardExpiryReady, markCardExpiryReady] = useState<boolean>(false);
  const [isCardCVCReady, markCardCVCReady] = useState<boolean>(false);
```

Reply:
or changing them to...
```suggestion
  const [cardNumberElement, setCardNumberElement] = useState<undefined | StripeElementBase>();
  const [cardExpiryElement, setCardExpiryElement] = useState<undefined | StripeElementBase>();
  const [cardCVCElement, setCardCVCElement] = useState<undefined | StripeElementBase>();
```
... as per my other comment.

Just a thought, not sure which is nicer.